index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>my portfolio</title>
+    <link rel="stylesheet" href="styling.css">
+    <link rel="stylesheet" media="(min-width: 768px)"  href="style.css">
+  </head>
+  <body>
+    <div class="container">
+        <header class="section1">
+          <nav class="navigation">
+            <div class="logo">
+              <a href="index.html" class="logo-text touch rough">my logo</a>
+            </div>
+            <section class="item-nav">
+              <a href="menu-items" class="menu-bar touch rough"><img src="mobile tag.png" alt="menu bar"></a>
+            </section>
+            <nav class="mail-button items">
+              <p class="mail touch rough"><img src="Union.png" alt="mail link"></p>
+              <p class="menu-items">
+                <section class="items touch rough">
+                  <p class="item1">portfolio</p>
+                  <p class="item2">about</p>
+                  <p class="item3">contact</p>
+                </section>
+              </p>
+            </nav>
+          </nav>
+        </header>
+      </section>
+      <main class="section2">
+        <article class="intro-text">
+          <p class="text1">Hey There.<br>I'm Chris.</p>
+          <p class="text2">I'm a Software Developer</p>
+          <p class="text3">I can help you build a product , feature <br> or
+            website. Look through some of my <br> work and experience! If you
+            like what you <br> see and have a project you need coded,<br>
+            don’t hesitate to contact me.</p>
+        </article>
+        <ul class="icons touch rough">
+          <li class="medium"><img src="medium.svg" alt="medium site"
+              class="icon-sizing"></li>
+          <li class="github"><img src="github.svg" alt="github site"
+              class="icon-sizing"></li>
+          <li class="angellist"><img src="angel list.svg" alt="angellist site"
+              class="icon-sizing"></li>
+          <li class="linkedn"><img src="linkedn.png" alt="linkedn site"
+              class="icon-sizing"></li>
+          <li class="twitter"><img src="twitter.png" alt="twitter site"
+              class="icon-sizing"></li>
+          </ul>
+        <article class="art">
+          <p class="flow"><img src="illustration-header-mobile.svg" alt="some
+              art" class="art-i"></p>
+        </article>
```

Comment:
- [ ] kindly consider adding this image as a background image as its a decoration this will increase your CSS skills on how to use an image as background image